src/main/MainScreen.js

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 import { StatusBar } from 'react-native';
 import Drawer from 'react-native-drawer';
 
+import { homeNarrow, specialNarrow } from '../utils/narrow';
 import { focusPing } from '../api';
 import Chat from '../chat/Chat';
 import MainNavBar from '../nav/MainNavBar';
 import StreamSidebar from '../nav/StreamSidebar';
-import UserListContainer from '../userlist/UserListContainer';
+import ConversationsContainer from '../conversations/ConversationsContainer';
 
 export default class MainScreen extends React.Component {
 
   componentDidMount() {
-    const { auth, fetchEvents, fetchUsersAndStatus, fetchMessages } = this.props;
+    const { auth, fetchEvents, fetchUsersAndStatus,
+      backgroundFetchMessages, fetchMessages } = this.props;
 
     fetchEvents(auth);
     fetchUsersAndStatus(auth);
-    fetchMessages(auth, Number.MAX_SAFE_INTEGER, 20, 0, [], [true, true]);
+    fetchMessages(auth, Number.MAX_SAFE_INTEGER, 20, 0, homeNarrow(), [true, true]);
+    backgroundFetchMessages(auth, Number.MAX_SAFE_INTEGER, 100, 0, specialNarrow('private'));
```

Comment:
@timabbott how hard would it be to add an API endpoint for recent private conversations? Seems cleaner than fetching private messages for that information.